### DISCUSSION TITLE
[helm-values] Generic solution by configuring image details

### DISCUSSION THREAD

**Discussion by ira-gordin-sap** [renovatebot/renovate#26733] [Suggest an Idea]
### Tell us more.

Can it be possible to configure the name of each field in [helm-values manager image details](https://docs.renovatebot.com/modules/manager/helm-values/#additional-information)?
For example metricbeat chart has the following parameters.
```
image: "docker.elastic.co/beats/metricbeat"
imageTag: "8.5.1"
```

**Comment by secustor** [comment]
No they logic is hard coded. 
https://github.com/renovatebot/renovate/blob/b6bef2121ed2516441bb99413f6934bb202aafff/lib/modules/manager/helm-values/util.ts#L8-L27

If you need to update different types of dependencies in `values.yaml` files use a [customManager](https://docs.renovatebot.com/modules/manager/regex/)

**Reply by rarkins** [reply]
We might be able to add this example to our "commonly used formats" though?

**Reply by secustor** [reply]
Sure, tough would need a contribution such as https://github.com/renovatebot/renovate/issues/15988

**Reply by ira-gordin-sap** [reply]
@secustor can I find some example for [customManager](https://docs.renovatebot.com/modules/manager/regex/) in some place?

**Reply by secustor** [reply]
There are some example behind the link you have posted, and you can find examples here in the discussions.

**Comment by ira-gordin-sap** [comment]
@secustor @rarkins I could not find something relevant in discussions. Can you please help?
